sktime/transformations/base.py

```diff
@@ -469,9 +469,18 @@ def transform(self, X, y=None, Z=None):
             Xt = convert_Panel_to_Series(Xt)
 
         if output_scitype == "Series":
+            # if the transformer outputs multivariate series,
+            #   we cannot convert back to pd.Series, do pd.DataFrame instead then
+            check_res = check_is_mtype(
+                ["pd.DataFrame", "pd.Series", "np.ndarray"], return_metadata=True
+            )
+            if not check_res[2]["is_univariate"] and X_input_mtype == "pd.Series":
```

Comment:
this may make it a bit more readable
```suggestion
            _, _, metadata = check_is_mtype(
                ["pd.DataFrame", "pd.Series", "np.ndarray"], return_metadata=True
            )
            if not metadata["is_univariate"] and X_input_mtype == "pd.Series":
```

Reply:
I've thought about it, but the first option is more stable against addition of further return args.
Though that may be too defensive, I agree your variant is more readable. Will change to it.